src/components/alert/alert.jsx

```diff
@@ -18,7 +18,7 @@ import ReachAlert from "@reach/alert";
 function Alert({ className, message, type, variant }) {
   return (
     <ReachAlert
-      className={`ddb-alert ddb-alert--${variant} ${className}`}
+      className={`dpl-alert dpl-alert--${variant} ${className}`}
```

Comment:
Because of the ErrorBoundary [here](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/feature/separat-opdateringtypescript-projekt-klargor-dpl-cms-DDFSOEG-90/src/core/mount.js#L30:L30). If we remove that we need to decide what to do instead.